__tests__/app.test.js

```diff
@@ -29,6 +29,70 @@ describe("/api/", () => {
     });
   });
   describe("/api/reviews", () => {
+    describe('"/api/reviews', () => {
+      test("200: responds with all reviews if no query is set in descending order", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then((reviews) => {
+            const body = reviews.body;
+
+            const keys = [
+              "review_id",
+              "title",
+              "category",
+              "designer",
+              "owner",
+              "review_body",
+              "review_img_url",
+              "created_at",
+              "votes",
+              "comment_count",
+            ];
+            expect(body.length === 13).toBe(true);
+            body.forEach((review) => {
+              expect(Object.keys(review)).toEqual(keys);
```

Comment:
`expect(Object.keys(review)).toEqual(keys);`

 this isn't the safest way to check if the object has the correct keys

 Objects are typically unordered, Arrays are ordered - if `Object.keys` returned keys in a different order this test would fail!

 Checking each key individually like you have in the categories test above would be better here